platform/commonUI/browse/src/BrowseObjectController.js

```diff
@@ -22,8 +22,9 @@
 /*global define,Promise*/
 
 define(
-    [],
-    function () {
+    ['../../../representation/src/gestures/GestureConstants', 
+        '../../edit/src/objects/EditableDomainObject'],
```

Comment:
odd multi-line array style.  should have open/close brace on separate lines, standard indents.